diff a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -239,8 +239,8 @@
 	  else if [ -x /bin/bash ]; then echo /bin/bash; \
 	  else echo sh; fi ; fi)

-HOSTCC       = gcc
-HOSTCXX      = g++
+HOSTCC       = gcc-4.9
+HOSTCXX      = g++-4.9
 HOSTCFLAGS   = -Wall -Wmissing-prototypes -Wstrict-prototypes -O2 -fomit-frame-pointer
 HOSTCXXFLAGS = -O2

@@ -326,7 +326,7 @@

 AS		= $(CROSS_COMPILE)as
 LD		= $(CROSS_COMPILE)ld
-CC		= $(CROSS_COMPILE)gcc
+CC		= $(CROSS_COMPILE)gcc-4.9
 CPP		= $(CC) -E
 AR		= $(CROSS_COMPILE)ar
 NM		= $(CROSS_COMPILE)nm
